lib/util/template/index.ts

```diff
@@ -161,9 +161,9 @@ function getFilteredObject(input: CompileInput): FilteredObject {
   for (const field of allAllowed) {
     const value = obj[field];
     if (is.array(value)) {
-      res[field] = value.map((element) =>
-        getFilteredObject(element as CompileInput)
-      );
+      res[field] = value
+        .filter(is.object)
```

Comment:
```suggestion
        .filter(is.plainObject)
```
Sorry, seen this function some lines below.